composer.json

```diff
@@ -24,13 +24,13 @@
         "symfony/console": "^2.3 || ^3.0",
         "symfony/doctrine-bridge": "^2.2 || ^3.0",
         "symfony/framework-bundle": "^2.2 || ^3.0",
-        "symfony/security-acl": "^2.2 || ^3.0",
+        "symfony/security-acl": "^2.2 || ^3.0^",
         "sonata-project/exporter": "^1.3.1",
         "sonata-project/admin-bundle": "^3.0",
         "sonata-project/core-bundle": "^3.0"
     },
     "require-dev": {
-        "simplethings/entity-audit-bundle": "~0.1",
+        "simplethings/entity-audit-bundle": "^0.1",
```

Comment:
`>= ^0.1 || <= ^0.9` is invalid. I think you meant `>= 0.1 || <= 0.9`.

Hyphen does the same AFAIK: `0.1 - 0.9`